scripts/jest/jest.config.js

```diff
@@ -1,26 +1,31 @@
+const ignores = ['node_modules', '__fixtures__', '__mocks__', 'dist', 'lib'];
+
 module.exports = {
     clearMocks: true,
-    restoreMocks: true,
+    collectCoverage: false,
+    collectCoverageFrom: ['src/**/*.ts', 'src/**/*.tsx', 'src/**/*.js', '!**/node_modules/**', '!**/__tests__/**'],
+    coverageDirectory: '<rootDir>/reports',
+    coveragePathIgnorePatterns: ['\\.stories.*$', 'src/icon/*', 'src/icons/*', 'src/illustration'],
+    globalSetup: '<rootDir>/scripts/jest/env-setup.js',
     moduleNameMapper: {
         'box-ui-elements-locale-data': '<rootDir>/i18n/en-US.js',
         'box-locale-data': '<rootDir>/node_modules/@box/cldr-data/locale-data/en-US',
         '\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$':
             '<rootDir>/scripts/jest/mocks/fileMock.js',
         '\\.(css|less|scss|md)$': '<rootDir>/scripts/jest/mocks/styleMock.js',
+        '^uuid$': require.resolve('uuid'),
```

Comment:
uuid errors were throwing without this, when uuid 9 is released then we could look into removing this but for now it works with this configuration